sanic/blueprints.py

```diff
@@ -17,11 +17,15 @@
         "name",
     ],
 )
-FutureListener = namedtuple("Listener", ["handler", "uri", "methods", "host"])
```

Comment:
not related, I like `attrs` over `namedtuple`.
And `namedtuple` is also slow when initializing, but doesn't matter in this case.